### ISSUE TITLE
Are there details about the generator？

### ISSUE THREAD

**Issue by 980202006** [yl4579/StarGANv2-VC#1]
Good job! Can you elaborate on the structure of the generator, including its layers and parameters

**Comment by 980202006** [comment]
whether to freeze F0 network

**Comment by 980202006** [comment]
how to modify the network to the unseen speaker

**Comment by yl4579** [comment]
I'm still working on cleaning our training code, but you can refer to [models.py](https://github.com/yl4579/StarGANv2-VC/blob/develop/models.py) for the architecture. Note that the whole architecture was directly taken from [stargan-v2](https://github.com/clovaai/stargan-v2) with only a few modifications.

The F0 network was frozen during training, so is the ASR network. As our framework is many-to-many, you'll need to either fine-tune the model for unseen speakers, or do some modifications on the style encoder and loss functions to make it a zero-shot conversion model.

**Comment by yl4579** [comment]
I have uploaded the pretrained models on VCTK dataset, please check and reopen this issue if you have any further questions.
